### PR TITLE
Added Social Media Icons

### PR DESCRIPTION
**Pull request by chinmaym07**
Hey @smilegupta Please Review . 

Screen Shot 2020-12-08 at 16 21 56
Screen Shot 2020-12-08 at 16 06 00
Screen Shot 2020-12-08 at 16 05 46

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -20,7 +20,7 @@
             href="https://fonts.googleapis.com/css2?family=Roboto+Mono:wght@500&family=Russo+One&display=swap"
             rel="stylesheet"
         />
-
+        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
```
Comment (smilegupta) [inline index.html]: Font awesome is not consumed in the project right,  wont it be better if we remove the cdn just to make code look clean

**Review by smilegupta — CHANGES_REQUESTED** [review]
Couple of suggestions, in now a days people also tend to have codepen and codesandbox also included on the profile. It will add the look and feel of the portfolio too.  
@chinmaym07 A suggestion from my end. As market demands to standout.

**Comment by chinmaym07** [comment]
Hey @smilegupta  I have done the changes .Please Review.

Screen Shot 2020-12-09 at 02 31 10
Screen Shot 2020-12-09 at 02 31 00

**Review by smilegupta — APPROVED** [review]

**Comment by smilegupta** [comment]
Good Work Done! You right awesome code.

**Comment by chinmaym07** [comment]
> Good Work Done! You right awesome code.

Thankyou So much @smilegupta :grinning: :+1:

**Pull request merged**